rest_api/tests/api_test/base.py

```diff
@@ -0,0 +1,160 @@
+
+class RestApiBaseTest(object):

```

Comment:
Please check with Amol & Ashish if the tests are to be run as dockerised containers. since other components (validator, TP, restAPI)  are run as containers it will be nice to run the tests inside container.

Reply:
Tests will run on the validation network without the dockers